src/components/earthEngine/EarthEngineLayer.js

```diff
@@ -0,0 +1,35 @@
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Checkbox } from '../core/index.js'
+import styles from './styles/EarthEngineLayer.module.css'
+
+const EarthEngineLayer = ({ layer, isAdded, onShow, onHide }) => {
```

Comment:
There is already a component named EarthEngineLayer (in src/components/map/layers). Is there a different name that could be used that would better represent what this component is?

Reply:
LayerTypeOption ?